lib/generator.ts

```diff
@@ -165,7 +165,7 @@ export async function generate(path: string, options?: GenerateOptions) {
   // Update rule doc for each rule.
   let initializedRuleDoc = false;
   for (const { name, description, schema } of details) {
-    const pathToDoc = join(path, pathRuleDoc).replace(/{name}/g, name);
+    const pathToDoc = join(path, pathRuleDoc).replace(/\{name\}/gu, name);
```

Comment:
similarly, why does the u flag help here? none of the chars in `{name}` can be part of a surrogate pair.

Reply:
I assume it's just for consistency so [require-unicode-regexp](https://eslint.org/docs/latest/rules/require-unicode-regexp) can be enabled?